doc/USER_GUIDE.md

```diff
@@ -0,0 +1,132 @@
+# Installation
+
+To configure a GitHub repository to use TACOS-GHA, you need to configure a
+number of workflows in the `.github/workflows/` directory in the repository.
+Examples of these workflow files can be found [here](workflows).
```

Comment:
This seems seriously unlikely to be kept up-to-date. I'd instead point them at the tacos-gha.demo workflows, which must necessarily be up to date.

Reply:
The demo repo can be in a state of flux, so pointing people there seems inadvisable. For now I prefer this.